provider/runbook_resource.go

```diff
@@ -96,6 +96,15 @@ func resourceRunbook() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 			},
+			"attachment_rule": {
+				Type:             schema.TypeString,
+				Optional:         true,
+				ValidateDiagFunc: validation.ToDiagFunc(validation.StringIsJSON),
+				StateFunc: func(value interface{}) string {
+					normalizedJSON, _ := structure.NormalizeJsonString(value)
+					return normalizedJSON
+				},
+			},
```

Comment:
nit: could this be moved up so that it is at the top of the optional args? i'm trying to keep everything following the pattern of required then optional then computed, all in alphabetical order